test_runner/regress/test_tenant_delete.py

```diff
@@ -509,20 +505,20 @@ def delete_tenant():
     def hit_remove_failpoint():
         env.pageserver.assert_log_contains(f"at failpoint {BEFORE_REMOVE_FAILPOINT}")
 
-    def hit_run_failpoint():
-        env.pageserver.assert_log_contains(f"at failpoint {BEFORE_RUN_FAILPOINT}")
+    def hit_run_failpoint(cursor=None):
+        env.pageserver.assert_log_contains(f"at failpoint {BEFORE_RUN_FAILPOINT}", offset=cursor)
 
     with concurrent.futures.ThreadPoolExecutor() as executor:
         background_200_req = executor.submit(delete_tenant)
         assert background_200_req.result(timeout=10).status_code == 202
 
         # Wait until the first request completes its work and is blocked on removing
         # the TenantSlot from tenant manager.
-        wait_until(100, 0.1, hit_remove_failpoint)
+        log_cursor = wait_until(100, 0.1, hit_remove_failpoint)
```

Comment:
the `hit_remove_failpoint` doesn't have a `return` statement, so, this `log_cursor` variable here will always be `None`

Seems not like what you intended

Reply:
Quite right, these assertions ended up as no-ops.  Fixed.